pegasus/sites.v3/hourofcode.com/i18n/es.yml

```diff
@@ -14712,7 +14712,7 @@ es:
     the help of AI and intelino Scratch! Create and train an AI model that recognizes
     signs. Then, use this model in intelino Scratch to program and control movement
     and steering of the intelino train.
-  tutorial_generationai_longdescription: Lánzate a esta aventura a través del tiempo para crear tus inventos con IA en el nuevo La hora del código de Minecraft: generación con IA. Aprende a resolver problemas, a ser creativo y a pensar computacionalmente para un uso ético de la IA. Con programación en bloques o Python, La hora del código enseña lo básico del código y cómo construir una IA mejor para todos.
+  tutorial_generationai_longdescription: 'Lánzate a esta aventura a través del tiempo para crear tus inventos con IA en el nuevo La hora del código de Minecraft: generación con IA. Aprende a resolver problemas, a ser creativo y a pensar computacionalmente para un uso ético de la IA. Con programación en bloques o Python, La hora del código enseña lo básico del código y cómo construir una IA mejor para todos.'
```

Comment:
@code-dot-org/platform -- I think this file is updated automatically in the sync. Is there a way to ensure this doesn't reappear after the next one?